docker/test-server-mysql-localci.yml

```diff
@@ -46,10 +46,10 @@ services:
         restart: always
         volumes:
             - type: bind
-              source: ${NGINX_PROXY_SSL_CERTIFICATE_PATH:-../src/test/cypress/certs/artemis-nginx+4.pem}
+              source: ${NGINX_PROXY_SSL_CERTIFICATE_PATH:-../src/test/playwright/certs/artemis-nginx+4.pem}
```

Comment:
how do the tests already run on playwright when the yml configuration files seem to still contain cypress?

Reply:
Test runs in CI are based on the build plans `docker/playwright-E2E-tests-*`. SSL certificates are currently not used in Playwright (no option to enable https authentication), but updated their paths nevertheless